python/doc/examples/reliability_sensitivity/estimate_moments_taylor.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
It would a good idea to remove "basic" from the initial sentence of the document "In this basic example..." like you did in other notebooks.

Reply:
You are right: 
3b0677171bb74a5159429957b8ffd4ee1850e052
I used this trick:
https://askubuntu.com/questions/84007/find-and-replace-text-within-multiple-files?newreg=438011e1fcec4fb2b0465dbe5b6b196f
```
find ./ -type f -readable -writable -name \*.ipynb -exec sed -i "s/In this basic example/In this example/g" {} \;
```
Fixed.